sources/core/Stride.Core.Design.Tests/TestPackageVersion.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+using System.Collections.Generic;
+
+using Xunit;
+using Stride.Core;
+
+namespace Stride.Core.Design.Tests
+{
+    public class TestPackageVersion
+    {
+        [Fact]
+        public void TestParseVersionsByValidVersionStrings()
+        {
+            var versionTests = new VersionTest[]
```

Comment:
You can change this `[Fact]` to `[Theory]`, remove the `VersionTest` structure and instead have the test accept the four values as arguments, then search replace each `new(...)` with `[InlineData(...)]` and put them under `[Theory]`. This greatly reduces the amount of code and allows running individual tests from Test Explorer.

Reply:
Thanks for the advice, have changed to this style.